data/test_domains/moodbot_domain.yml

```diff
@@ -0,0 +1,32 @@
+%YAML 1.1
```

Comment:
is this domain different from the one in `examples/moodbot`?

Reply:
No

Reply:
this should be removed if it is just a duplicate